lib/rules/no-multi-spaces.js

```diff
@@ -12,7 +14,17 @@
 module.exports = function(context) {
 
     // the index of the last comment that was checked
-    var lastCommentIndex = 0;
+    var exceptions = { "Property": true },
+        exceptionCount = 1,
+        options = context.options[0],
+        lastCommentIndex = 0;
+
+    if (options && options.exceptions) {
+        Object.keys(options.exceptions).forEach(function (key) {
+            exceptions[key] = options.exceptions[key];
+        });
+        exceptionCount = Object.keys(exceptions).length;
```

Comment:
I don't think `exceptionCount` is needed. There's always at least one entry in exceptions, so this is always truthy.